examples/player/simid_player.html

```diff
@@ -50,11 +50,50 @@
       simidPlayer.initializeAd();
     }
 
+    /**Disables or enables stop, skip, and fatal error buttons
+     *  depending on if the ad is showing.
+     * @param {boolean} disable If buttons should be disabled or enabled
+     */
+    function setButtonState(disable) {
+      let adRequests = document.querySelectorAll(".ad_request");
+      adRequests.forEach((request) => {
+        request.disabled = disable;
+      });
+    }
+
     function playAd() {
       if (!simidPlayer) {
         initAd();
       }
       simidPlayer.playAd();
+      setButtonState(/* disable= */ false);
+    }
+
+    function closeAd() {
+      if (!simidPlayer) {
+        return;
+      }
+      simidPlayer.stopAd();
+      simidPlayer.sendLog("Ad stopped by player");
```

Comment:
what does send log do?  It might not be too relevant to send a log to the creative if it already knows its been stopped.

Reply:
This log is meant to distinguish between what action was actually sent by the player since Stop, Fatal Error, and Skip all shut down the ad. Let me know if you think I should get rid of the logs, I put them there to be informative

Reply:
Julia and I discussed this offline and I will leave it up to her.